evaluator/images/dotnet/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM kelvin/base
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    dotnet6=6.0.108-0ubuntu1~22.04.1 \
+    aspnetcore-runtime-6.0=6.0.108-0ubuntu1~22.04.1 \
+    python3-pip
```

Comment:
it may be good idea to remove apt caches - they are not usable after some time in following commands anyway
`&& rm -rf /var/lib/apt/lists/*`